Modules/Forum/classes/class.ilObjForum.php

```diff
@@ -407,6 +410,70 @@ public function cloneObject(int $target_id, int $copy_id = 0, bool $omit_tree =
 
         return $new_obj;
     }
+    public function updateUserNotifications(string $notification_type): void
```

Comment:
```suggestion

    public function updateUserNotifications(string $notification_type): void
```

One addition: Can you prepare an `Enum` for `trunk` (only) instead of passing a string for the `Notification Type`?